### PR TITLE
JSON data type support

### PR DESCRIPTION
**Pull request by osheroff**
This is chained atop #524.  Provides JSON column support under 5.7.  Only with the binlog-connector library. 

@zendesk/rules

### CODE REVIEW

**Comment by dadah89** [comment]
Logic seems fine to me. Can you add some tests? (ColumnDef and RowMap toJson maybe?)

File: src/main/java/com/zendesk/maxwell/schema/columndef/JsonColumnDef.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		try {
+			String jsonString = JsonBinary.parseAsString((byte[]) value);
+			return new RawJSONString(jsonString);
+		} catch (IOException e) {
+			throw new RuntimeException(e);
+		}
+	}
+
+	@Override
+	public String toSQL(Object value) {
```
Comment (dadah89) [inline src/main/java/com/zendesk/maxwell/schema/columndef/JsonColumnDef.java]: Is this not used?

Comment (osheroff) [inline src/main/java/com/zendesk/maxwell/schema/columndef/JsonColumnDef.java]: it's not, yeah.  It's old, dangling code that I'm somewhat loathe to get rid of.  But maybe i should just get rid of it.

**Comment by osheroff** [comment]
re: tests.  yes.  I swear I had already written them.

**Comment by osheroff** [comment]
ok tests added

**Pull request merged**